activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+*   Ensure `ActiveSupport::Testing::Isolation::Forking` closes pipes
+
+    Internally, `Isolation` uses the `Forking` module to provide test isolation
+    in environments supporting forking processes, which uses `IO.pipe` to send
+    results from the fork back to the parent.
+
+    Previously, the non-block version of `IO.pipe` was used, and the "read" end
+    of the pipe was not closed in the parent process. Now, the block version of
+    `IO.pipe` is used, which ensures both ends of the pipe are closed after the
+    block returns.
+
+    *Sam Bostock*
```

Comment:
I originally did not add a changelog entry, but I noticed that [`ActiveSupport::Testing::Isolation::Forking`](https://api.rubyonrails.org/classes/ActiveSupport/Testing/Isolation/Forking.html) is documented, so I figured maybe I should.

Reply:
Sorry for the delay – I've updated the changelog to the above, but in the past tense, so as to describe the previous behavior, as opposed to the PR description describing the existing bug.